external/opal-prd/hostboot-interface.h

```diff
@@ -71,6 +84,61 @@ enum MemoryError_t
 #define HBRT_PM_LOAD    0
 #define HBRT_PM_RELOAD  1
 
+/** Common return codes for scom_read(), scom_write(). */
+#define HBRT_RC_RANGE__scom   0x1000
+
```

Comment:
Better s/scom/SCOM/ .. Generally we prefer upper case letter for macros.

Reply:
Sure, I can do that